Dockerfile

```diff
@@ -1,6 +1,9 @@
 # syntax=docker/dockerfile:1
 FROM --platform=linux/amd64 synthetixio/docker-e2e:18.16-ubuntu as base
 
+# Prefer IPv4 for DNS resolution
```

Comment:
Sounds like some serious detective work! Consider explaining in the comment _why_ to prefer IPv4.